content/guides/payment-terminals.md

```diff
@@ -0,0 +1,60 @@
+---
```

Comment:
There's a comment in the [Migrating remaining legacy guides](https://www.notion.so/centrapay/Migrating-remaining-legacy-guides-a15efbe41db04386937ea16862264220) Notion meeting notes page about redirecting to this page on the `/guides/merchant-terminals` path. 

I think we should do something like this: https://medium.com/dont-leave-me-out-in-the-code/nuxt-redirects-in-4-simple-steps-9a0d5cb4c0d

I also think the redirect url should be defined in the front matter on each page.

Let me know if you need a hand with this :)

Reply:
I think this task should be in a follow up PR. ;)

Reply:
If it's a seperate PR then the `guides/merchant-terminals` path will be broken. That path existed on the legacy guides. We don't want to break any existing links :)

Reply:
@Wulingoh Yes PRs should be small but they should encapsulate a single functional change end-to-end 👍 

As Raymond mentioned - not breaking links for existing users is part of our requirements here. 

I'd suggest a separate PR containing all the work required to migrate the `merchant-terminals` guide (my preferred option would have been small, separate PRs for **each** guide)

Reply:
As per discussion with @tom-blake and @Ray2126, I have reverted the changes for `merchant-terminals.md` and will remain in the legacy folder. Meanwhile, I have deleted the `payment-terminals.md` from Nuxt content guides folder.

Reply:
You should update the description :)